scout/adapter/mongo/variant.py

```diff
@@ -832,6 +832,27 @@ def evaluated_variants(self, case_id, institute_id):
         # Return a list with the variant objects
         return variants.values()
 
+    def get_variant_file(self, case_obj, category, variant_type):
+        """Retrieve that should be parsed to extract variants
+
+        Accepts:
+            case_obj(dict): a case dictionary
+            category(str): "snv", "sv", "str", "cancer"
+            variant_type(str): "clinical" or "research"
+
+        Returns:
+            variant_file(str): string path to a VCF file on disk
+        """
+        variant_file = None
+        for var_type in ["snv", "sv", "str", "research"]:
```

Comment:
```suggestion
        for var_type in ["snv", "sv", "str", "cancer"]:
```

Reply:
I dunno, I might have chosen a different temp variable name for the category match as well; var_type to me sounds more like what would be matched to variant_type. But everything works! 😊

Reply:
We can change it, what do you propose?

Reply:
Ah yes, nasty typo!